dnsdist: use luajit on luajit archs except aarch64; lua on other archs

killgufo opened the pull request:
Workaround a luajit runtime SIGILL error on aarch64 by using lua

killgufo committed:
dnsdist: use luajit on luajit archs except aarch64; lua on other archs

Workaround a luajit runtime SIGILL error on aarch64 by using lua

Signed-off-by: Daniele Bonomi <github@notmycloud.com>

BKPepe commented:
Why the previosu PR was closed - https://github.com/openwrt/packages/pull/27873 and opened a new one?

Habbie commented:
> If @Habbie agrees, being maintainer both of dnsdist and the package for openwrt.

hello!

>  dnsdist: use luajit on luajit archs except aarch64; lua on other archs

this is the right goal. Somebody will have to judge that the syntax in this PR does that.

killgufo commented:
The validation of the logic I implemented shouldn't be too difficult. 
I implemented a simple if-then-else construct that matches the subject of this PR.
Who can do the validation now that we have the blessing of the maintainer?

BKPepe commented:
> Who can do the validation now that we have the blessing of the maintainer?

Well, answer to the questions, which were in your previous PR - https://github.com/openwrt/packages/pull/27873#issuecomment-3534673652

killgufo commented:
@Habbie what's your view?
